IDE/GitISPFClient/sbgzexec/bgzdbbub.rexx

```diff
@@ -1,313 +1,659 @@
+       Do
+         DoReq = 1
+
+         /* Need to load a temp table with the props for this repo */
+         'CONTROL ERRORS RETURN'
+         'TBEND    TMPPROPS'
+         'CONTROL ERRORS CANCEL'
+         'TBCREATE TMPPROPS',
+           'KEYS(BGZUBREP,BGZUBLOC,BGZPROW)',
+           'NAMES(BGZPNAME,BGZPVAL,BGZPMCMD) WRITE'
+         'TBOPEN BGZPROPS'
+         /* If it didn't exist yet, create it. */
+         If RC = 8 Then
+         Do
+           'TBCREATE BGZPROPS',
+           'KEYS(BGZUBREP,BGZUBLOC,BGZPROW)',
+           'NAMES(BGZPNAME,BGZPVAL,BGZPMCMD) WRITE'
+         End
+         'TBSORT BGZPROPS FIELDS(BGZPROW)'
+         'TBTOP  BGZPROPS'
+         /* Need to delete all the -- properties and rebuild from BGZDBBUB */
+         'TBSKIP BGZPROPS'
+         TB_RC = rc
+         Do While (TB_RC = 0)
+           If BGZPNAME = '--sourceDir' |,
+              BGZPNAME = '--workDir'   |,
+              BGZPNAME = '--hlq'       |,
+              BGZPNAME = '--application' Then
+           Do
+             'TBDELETE BGZPROPS'
+           End
+           'TBSKIP BGZPROPS'
+           TB_RC = rc
+         End
+
+         BGZUBREP = BGZUSREP
+         BGZUBLOC = BGZUSLOC
+         /* Now add them back in */
+         BGZPROW = '000001'
+         BGZPNAME = '--sourceDir'
+         BGZPVAL  = sandbox
+         BGZPMCMD = ''
+         'TBADD BGZPROPS ORDER'
+         BGZPROW = '000002'
+         BGZPNAME = '--workDir'
+         BGZPVAL  = workdir
+         BGZPMCMD = ''
+         'TBADD BGZPROPS ORDER'
+         BGZPROW = '000003'
+         BGZPNAME = '--hlq'
+         BGZPVAL  = hlq
+         BGZPMCMD = ''
+         'TBADD BGZPROPS ORDER'
+         BGZPROW = '000004'
+         BGZPNAME = '--application'
+         BGZPVAL  = appname
+         BGZPMCMD = ''
+         'TBADD BGZPROPS ORDER'
+
+         'TBTOP  BGZPROPS'
+         'TBSKIP BGZPROPS'
+         TB_RC = rc
+         Do While (TB_RC = 0)
+           If BGZUBREP = BGZUSREP Then
+             'TBADD TMPPROPS ORDER'
+
+           'TBSKIP BGZPROPS'
+           TB_RC = rc
+         End
+         'TBSORT TMPPROPS FIELDS(BGZPROW)'
+         'TBTOP  TMPPROPS'
+         'TBSKIP TMPPROPS'
+         Do While(RC = 0)
+           shellcmd=shellcmd || '' BGZPNAME BGZPVAL
+           'TBSKIP TMPPROPS'
+         End
+         'TBCLOSE BGZPROPS'
+         'TBEND   TMPPROPS'
+
+         shellcmd=shellcmd || userBuildCmd
+
+         'TBMOD BGZDBBUB'
+         BGZFLOG = BGZBLWRK'/dbbub.log'
+         'VPUT (BGZFLOG) SHARED'
+         DBB_rc = BGZCMD('dbbub' shellcmd)
+         BGZCMD = ''
+       End
+       'REMPOP'
+       /* View build output on completion  */
+       If BGZBLLOG = '/' & DoReq = 1 Then
+       Do
+         x = Lastpos('.',BGZUSFIL)
+         filename = Substr(BGZUSFIL,1,x-1)
+         ext      = Substr(BGZUSFIL,x+1)
+         input= filename
+         input_upper = translate(input, uppercase, lowercase)
+         filename = input_upper
+         BGZUSLOG = filename'.'ext'.log'
```

Comment:
@campagboy This file extension in the user build log seems to be custom to your version of the Git ISPF client. It opened an empty file on my side:

![image](https://github.com/user-attachments/assets/d578983e-2040-4c58-93e3-a2ec4452bfb2)